lib/screens/signup_screen.dart

```diff
@@ -3,88 +3,97 @@ import 'package:furniture/commons/reusable_textfield.dart';
 import 'package:furniture/commons/reusable_button.dart';
 import 'package:furniture/screens/login_screen.dart';
 
-class SignupScreen extends StatelessWidget {
+class SignupScreen extends StatefulWidget {
   const SignupScreen({super.key});
 
   @override
-  Widget build(BuildContext context) {
-    Size size = MediaQuery.of(context).size;
-    GlobalKey<FormState> formKey = GlobalKey<FormState>();
-    TextEditingController password = TextEditingController();
-    TextEditingController confirmPassword = TextEditingController();
-    String? confirmPasswordValidator(
-        String? password, String? confirmPassword) {
-      if (confirmPassword == null || confirmPassword.isEmpty) {
-        return 'Confirm Password is required';
-      }
-
-      if (password != confirmPassword) {
-        return 'Passwords do not match';
-      }
-
-      return null; // Passwords match
+  State<SignupScreen> createState() => _SignupScreenState();
+}
+
+class _SignupScreenState extends State<SignupScreen> {
+  GlobalKey<FormState> formKey = GlobalKey<FormState>();
+
+  TextEditingController password = TextEditingController();
+
+  TextEditingController confirmPassword = TextEditingController();
```

Comment:
rename to passwordCtrl
rename to confirmPasswordCtrl,

so when reading code down below, we can understand its controller